client/src/main/java/io/pravega/client/stream/impl/Controller.java

```diff
@@ -272,4 +271,11 @@
     @Override
     void close();
 
+    /**
+     * Refreshes an expired/non-existant delegation token.
+     * @param scope         Scope of the stream.
+     * @param streamName    Name of the stream.
+     * @return
+     */
+    CompletableFuture<String> getOrRefeshDelegationTokenFor(String scope, String streamName);
```

Comment:
There's a little spelling mistake in there.

Reply:
Fixed.